test-harness/local-run-wo-docker.sh

```diff
@@ -6,6 +6,11 @@
 
 . ./test-harness/init.sh --source-only
 
+
+if [ -f ~/.bash_profile ]; then
```

Comment:
why do we need to source `bash_profile`?

Reply:
Looks like this is left over from when I was working with Joe Long on making BashWizard more universal. It appears this code got snuck in under my radar. Removed